regression/cbmc-smt/union1/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+main.c
+--smt2
+activate-multi-line-match
+^EXIT=10$
```

Comment:
Maybe we should also have at least one test that actually verifies successfully?